Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -490,21 +490,25 @@ theorem norm_nonneg' (a : E) : 0 ≤ ‖a‖ := by
 #align norm_nonneg' norm_nonneg'
 #align norm_nonneg norm_nonneg
 
-/- porting note: meta code, do not port
-section
+namespace Mathlib.Meta.Positivity
 
-open Tactic Tactic.Positivity
+open Lean Meta Qq Function
 
 /-- Extension for the `positivity` tactic: norms are nonnegative. -/
```

Comment:
Would be good if the comment indicated that this was just the *multiplicative* case. Maybe:
```lean
/-- Extension for the `positivity` tactic: multiplicative norms are nonnegative, via `norm_nonneg'`. -/
```

Reply:
```suggestion
/-- Extension for the `positivity` tactic: multiplicative norms are nonnegative, via `norm_nonneg'`. -/
```